picard/ui/widgets/scripttextedit.py

```diff
@@ -175,6 +179,19 @@ def get_selected(self):
         return self.last_selected
 
 
+def _clean_text(text):
+    return "".join(_replace_control_chars(text))
+
+
+def _replace_control_chars(text):
+    import unicodedata
+    for ch in text:
+        if ch not in ('\n', '\r', '\t') and unicodedata.category(ch)[0] == "C":
```

Comment:
IMHO, `('\n', '\r', '\t')` should be stored in a variable outside the loop, and made a set:

```python
simple_ctrl_chars = {'\n', '\r', '\t'}
for ch in text:
  if ch not in simple_ctrl_chars ...
```

Reply:
Yes, that reads better. Changed it